erpnext/projects/doctype/project/project.py

```diff
@@ -322,17 +322,42 @@ def update_sales_amount(self):
 		self.total_sales_amount = total_sales_amount and total_sales_amount[0][0] or 0
 
 	def update_billed_amount(self):
+		# nosemgrep
+		self.total_billed_amount = (
+			self.get_total_billed_amount_from_parent() + self.get_total_billed_amount_from_child()
+		)
+
+	def get_total_billed_amount_from_parent(self):
+		# nosemgrep
+		total_billed_amount = frappe.db.sql(
+			"""select sum(base_net_amount)
+			from `tabSales Invoice Item` si_item, `tabSales Invoice` si
+			where si_item.parent = si.name
+				and (si.project = %s)
+				and si.docstatus=1
+				and si.project is not null""",
```

Comment:
if `si.project` is compared against something 2 line above, it obviously can't be `null` here.